domains/luisalejandrobf.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "luisalejandrobf",
+    "email": "luisalejandrobravoferreira@gmail.com"
+  },
+  "record": {
+    "TXT": "google-site-verification=sGmOAXQM9W1XepIo-CgWM_GlqEMW7z96P10kOC2p6fs"
```

Comment:
?
Please read the docs on how to use Vercel with the domain https://is-a.dev/docs/vercel/

Reply:
Okay, sorry, I didn't know. I just read the docs, but I wanted to ask: If I use the CNAME or A record methods, will I be able to include a TXT record so I can try and optimize SEO a little with google search console? Thanks!

Reply:
Yes, should be fine if you are using it with A records. With CNAME no since you can't add other records with it

Reply:
Sure, thank you